### ISSUE TITLE
Building Gadgets Building Gadget is missing Building preview.

### ISSUE THREAD

**Issue by Moleculor** [NillerMedDild/Enigmatica2Expert#1313]
**Modpack version**
E2:E 1.67

**Can the issue be reproduced?**
Yes

The Building Gadget item from Building Gadgets is missing its preview. 

[Fixed in Building Gadgets v2.7.2.](https://github.com/Direwolf20-MC/BuildingGadgets/issues/329)

**Comment by NillerMedDild** [comment]
Thanks for the report.

**Comment by Frikkle** [comment]
Aren't we on 2.7.2 though? I looked into this two weeks ago and I remember seeing the fix on their tracker but noticed we are already on that version.

Never got around to commenting on it properly and submitting a detailed report

**Comment by Moleculor** [comment]
Unless 2.7.2 is only in a beta version, and/or somehow I've screwed up an update somewhere, and/or someone has time travel, no?

The Twitch client lists Building Gadgets 2.7.1 in the 1.67 version, which is the current release version. My MultiMC instance only had 2.7.1 in it before I updated to the latest version of Building Gadgets manually. 

2.7.2 of Building Gadgets was only released on August 9th. The E2:E Curseforge page shows the most recent files are all from July.
